src/api/subscriptions.js

```diff
@@ -219,6 +226,15 @@ export let OffersRequest;
 export let LoginRequest;
 
 
+/**
+ * @typedef {{
+ *   token: (string|undefined),
+ *   authCode: (string|undefined),
+ * }}
+ */
+export let AutoLoginRequestCallback;
```

Comment:
Do you want to call this a callback if its just a value? Why not use AutoLoginRequest.

Reply:
done